File: inc.graphql-resources/backend/nodejs/models/core/admin_level.js
module.exports = (sequelize, DataTypes) => {
    const AdminLevel = sequelize.define('admin_level', {
        admin_level_id: {
            type: DataTypes.STRING(40),
            primaryKey: true,
            allowNull: false
        },
        name: {
            type: DataTypes.STRING(100),
            allowNull: true
        },
        special_access: {
            type: DataTypes.BOOLEAN,
            defaultValue: 0
        },
        sort_order: {
            type: DataTypes.INTEGER,
            defaultValue: 0
        },
        default_data: {
            type: DataTypes.BOOLEAN,
            defaultValue: 0
        },
        active: {
            type: DataTypes.BOOLEAN,
            defaultValue: 1
        }
    }, {
        tableName: 'admin_level',
        timestamps: false
    });

    AdminLevel.associate = (models) => {
        AdminLevel.hasMany(models.admin, {
            foreignKey: 'admin_level_id'
        });
    };

    return AdminLevel;
};

File: inc.graphql-resources/backend/python/models/core/admin_level.py
from sqlalchemy import Column, String, Integer, Boolean
from database import Base

class AdminLevel(Base):
    __tablename__ = 'admin_level'

    admin_level_id = Column(String(40), primary_key=True)
    name = Column(String(100), nullable=False)
    sort_order = Column(Integer, default=0)
    active = Column(Boolean, default=True)